api/proto/info/info.proto

```diff
@@ -535,24 +545,43 @@ message ContentResources {
   uint64 curSizeBytes = 1;  // Current disk usage
 }
 
+// ZInfoContentTree is used to describe the top of a tree, which has
+// a UUID (and displayName) from the config ContentTree message.
+// The components (or blobs) used by that are referenced by the compoentShaList
+// and in a separate ZInfoBlobList info message
 message ZInfoContentTree {
   string uuid = 1;
   string displayName = 2;
   string sha256 = 3;
   ContentResources resources = 4;
-  UsageInfo usage = 5;  // Same info as for volumes
-  repeated string componentShaList = 6;
+  UsageInfo usage = 5;
+  ZSwState state = 6;
+  uint32 downloadProgress = 7;
```

Comment:
Rename 'downloadProgress' to 'progressPercentage'. As we discussed offline: it might make sense to have a single field whose meaning is dependent on the state. Thus if DOWNLOAD_STARTED it means the download progress, if DELIVERED (and not INSTALLED) it means the verification progress, etc.

Reply:
Yeah I can do that as long as you're prepared to do the rename on the cloud side.  I'll do that for all the Progress fields.

Note that in EVE (as part of that separate PR 975) I introduced a new state for VERIFYING (plus renamed DELIVERED to VERIFIED), but my gut feel is that adding that one more state in the proto file isn't worth-while. But we could potentially rename the existing ones.
In EVE's definition I have DOWNLOADING (instead of DOWNLOAD_STARTED_ and VERIFIED (instead of DELIVERED).

We can do this rename cleanup separately. I note that prototool in yetus complains about the upper case enum values (which I'm currently ignoring) but if we want to address that we might as well make the enum values more clear?  But that is a separate "rename" task in my mind.